### ISSUE TITLE
Editor TypeScript Error

### ISSUE THREAD

**Issue by PaitoAnderson** [telerik/kendo-react#201]
Getting an error when importing the new HTML Editor into a TypeScript project.

<img width="670" alt="screen shot 2019-03-02 at 1 54 50 pm" src="https://user-images.githubusercontent.com/4108718/53686370-16941100-3cf4-11e9-83e8-1f454b26ba33.png">

Side note, I also added these two packages to resolve other type errors:
"@types/prosemirror-state": "1.2.3",
"@types/prosemirror-view": "1.3.1",

## I'm submitting a...
* Bug report

## Current behavior
```
ERROR in [at-loader] ./node_modules/@progress/kendo-react-editor/dist/npm/Editor.d.ts:3:10 
          TS2440: Import declaration conflicts with local declaration of 'EditorState'.
```

## Expected behavior
Build without error

## Minimal reproduction of the problem with instructions
`import { Editor } from '@progress/kendo-react-editor';`

## Environment

Package versions:
```
├── @progress/kendo-data-query@1.5.0
├── @progress/kendo-drawing@1.5.8
├── @progress/kendo-react-buttons@2.7.1
├── @progress/kendo-react-charts@2.7.1
├── @progress/kendo-react-dateinputs@2.7.1
├── @progress/kendo-react-dialogs@2.7.1
├── @progress/kendo-react-dropdowns@2.7.1
├── @progress/kendo-react-editor@2.7.1
├── @progress/kendo-react-grid@2.7.1
├── @progress/kendo-react-inputs@2.7.1
├── @progress/kendo-react-intl@2.7.1
├── @progress/kendo-react-layout@2.7.1
├── @progress/kendo-react-pdf@2.7.1
├── @progress/kendo-theme-bootstrap@3.2.2
├── @types/bootstrap@4.2.2
├── @types/draft-js@0.10.28
├── @types/history@4.7.2
├── @types/jquery@3.3.29
├── @types/moment-timezone@0.5.10
├── @types/prop-types@15.7.0
├── @types/prosemirror-state@1.2.3
├── @types/prosemirror-view@1.3.1
├── @types/react@16.8.6
├── @types/react-dom@16.8.2
├── @types/react-redux@5.0.20
├── @types/react-router@4.0.32
├── @types/react-router-dom@4.3.1
├── @types/react-router-redux@5.0.15
├── @types/webpack-env@1.13.8
├── acorn@6.1.1
├── aspnet-webpack@3.0.0
├── awesome-typescript-loader@5.2.1
├── babel-core@6.26.3
├── bootstrap@4.3.1
├── clean-webpack-plugin@1.0.1
├── css-loader@2.1.0
├── draft-js@0.10.5
├── draft-js-export-html@1.3.0
├── exceptionless@1.6.1
├── hammerjs@2.0.8
├── jquery@3.3.1
├── lodash@4.17.11
├── mini-css-extract-plugin@0.5.0
├── moment-timezone@0.5.23
├── popper.js@1.14.7
├── react@16.8.3
├── react-document-title@2.0.3
├── react-dom@16.8.3
├── react-ga@2.5.7
├── react-hot-loader@4.7.1
├── react-redux@5.0.7
├── react-router-dom@4.3.1
├── react-router-redux@5.0.0-alpha.9
├── redux@3.7.2
├── redux-thunk@2.2.0
├── rimraf@2.6.3
├── terser-webpack-plugin@1.2.2
├── ts-debounce@1.0.0
├── typescript@3.3.3
├── webpack@4.29.6
├── webpack-cli@3.2.3
├── webpack-dev-middleware@3.6.0
├── webpack-hot-middleware@2.24.3
└── webpack-merge@4.2.1
```

Browser:

- Chrome (desktop) version 72.0.3626.121

System:
- TypeScript version: 3.3.3
- Node version: v11.10.0 
- NPM Version: v6.8.0
- Platform:  macOS

**Comment by Xizario** [comment]
The error says that you have declared something locally in the same file with the name `EditorState` and in the same time you are importing it with the same name. Isn't this the case or I am missing something?

**Comment by PaitoAnderson** [comment]
Correct.

It's a file in your package though:
/node_modules/@progress/kendo-react-editor/dist/npm/Editor.d.ts

**Comment by nstoychev** [comment]
Fixed in 2.8.0-dev.201903081502 version. Please refer to the following article about obtaining the development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

**Comment by Xizario** [comment]
Fixed in `2.8.0`
